.eleventy.js

```diff
@@ -1 +1,8 @@
 require('dotenv').config();
+
+module.exports = function(eleventyConfig) {
+  eleventyConfig.addPassthroughCopy('css')
+  return {
```

Comment:
❓Do you need this return statement here? I think you enable `passthrough` with the line above already?

Reply:
Just tried it and I don't need it. I'll cut it.